andela/vale-ah-frontend
# [WIP] #165182886 Get Single Recipe Feature

#### What does this PR do?
Add fetching a single recipe by slug functionality
#### Description of Task to be completed?
- setup the redux actions and reducer
- create `SingleRecipe` component
- create subcomponents (`Banner`, `IngredientList`, `StepList` ...)
- connect `SingleRecipe` component to the redux store
- write unit and integration tests
#### How should this be manually tested?
- ensure that the build passes on Travis
- check the review app and make sure everything works as expected
#### Any background context you want to provide?
Recipes form the backbone of our application and introduce us to CRUD which is the building block of most web applications.
#### What are the relevant pivotal tracker stories?
[#165182886](https://www.pivotaltracker.com/story/show/165182886)
#### Screenshots (if appropriate)
#### Questions:
